ee/modules/380-metallb/openapi/config-values.yaml

```
@@ -167,6 +167,7 @@ properties:
           type: string
           description: |
             The name of the pool (you can specify it using the service annotation as follows: `metallb.universe.tf/address-pool: <name>`).
```

```suggestion
            The name of the pool (you can specify it using the service annotation as follows: `metallb.universe.tf/address-pool: <name>`). It should conform to RFC 1123: lowercase, consists of alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character.
```

Also update description in doc-ru-config-values.yaml and validation job will pass (https://github.com/deckhouse/deckhouse/actions/runs/3530783527/jobs/5923168036#step:4:18).